ios/RNSScreenStack.mm

```diff
@@ -752,7 +752,7 @@ - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
   [self cancelTouchesInParent];
   return YES;
 #else
-  // RNSPanGestureRecognizer will receive events iff topScreen.fullScreenSwipeEnabled == YES;
+  // RNSPanGestureRecognizer will receive events if topScreen.fullScreenSwipeEnabled == YES;
```

Comment:
The `iff` is intentional - it is widely used, especially in mathematics & stands for `if and only if` (`wtedy i tylko wtedy`) 😄 

```suggestion
  // RNSPanGestureRecognizer will receive events iff topScreen.fullScreenSwipeEnabled == YES;
```

Reply:
Ah, haven't heard about that one before. That's good to know, thanks for enlightening me! 😃 I will revert that change